Add pre-intro for Unit 12 Ethics and Bias (#86)
johko/computer-vision-course: chapters/en/Unit 12 - Ethics and Biases/pre-intro.mdx

# Unit 12. Ethics and Bias in Computer Vision: Exploring Ethical Foundations in CV Models

> Welcome to the Ethics and Bias unit of our Computer Vision Course! 📸✨ This segment is designed to unravel the critical elements of ethics and bias within the realm of computer vision.

Author: Sure, it has been quite a time since I left my PhD program, but I think the academic style of writing is still with me 😅 
I will take a look at the content once again and make some more changes to make the style simpler.

## What you’ll learn 🖼️🤖

> In this unit we'll understand the ethical dimensions and potential biases within AI models, and how it is crucial for responsible development and deployment of computer vision systems. 

Reviewer: ```suggestion
In this unit we'll understand the ethical dimensions and potential biases within AI models and how it is crucial for responsible development and deployment of computer vision systems. 
```

A brief outline of the unit is given below:

> - We will start off this unit with Chapter 1, where we discuss the implications of the famous ImageNet Roulette Case Study.

Reviewer 2: Just a small wording suggestion from my side, but feel free to ignore, if you like it better the other way :wink: 

```suggestion
- We will start off this unit with Chapter 1, where we discuss the implications of the popular ImageNet Roulette Case Study.
```

Author: Sure @johko popular sounds better I will change it 🤗

Author: Done :D

- In Chapter 2, we'll discuss about the ethical considerations intertwined with AI and computer vision technologies and why it is important to be fair, while developing CV systems.

> - In Chapter 3, we'll learn how bias can infiltrate AI models across various modalities such as text, vision, and speech.

Reviewer 3: Shouldn't we focus on images only here? Maybe text too, but I don't see how audio belongs here.

Author: Yes, we are covering a case study being worked on by one of our team members @Zekrom-7780 that is why we included audio. We might keep it focused to vision + text as suggested.

Reviewer 4: Yes, so Basically I was working on a case study which delves into the impact of audiobook narration on speech recognition software. 
- The predominance of white male narrators in audiobooks has resulted in a bias within the training data for speech recognition algorithms. 
- As a consequence, users from different socio-economic or ethnic backgrounds experience underperformance when utilizing this software, highlighting the need for more diverse training datasets to ensure accuracy across diverse user demographics, 
- And I felt that this would be a very interesting topic 

What do you think @MKhalusova ??

Reviewer 3: It is absolutely a very interesting topic, and we can probably briefly mention it, but I would suggest trying to keep the main focus on CV

Author: Sure @MKhalusova we will keep our focus on CV, thank you for your feedback 😄

- In Chapter 4, we'll discuss various types of biases and their implications on computer vision models.
- In Chapter 5, we'll discuss different ways to spot bias, and metrics for evaluating biases in CV models with the help of practical case studies.
- In Chapter 6, we'll learn about the strategies and methods to mitigate biases specifically within computer vision models.
- Finally, we close with Chapter 7 and discuss HuggingFace's mission and initiatives towards fostering ethical AI for society.

## Journey through the unit 🏃🏻‍♂️🏃🏻‍♀️

> Let's embark on a journey that merges theoretical foundations, practical case studies, and ethical contemplations inherent in the landscape of computer vision. From exploring real-world examples like the ImageNet Roulette case study to evaluating biases in AI models recognizing "Gay Face", Twitter's Saliency Algorithm and similar case studies, this unit delves deep into understanding, assessing, and mitigating biases in computer vision systems.

Reviewer 3: Again, I would urge you to simplify the writing style.

Author: Sure I will take a look once again and simplify the style. Thank you @MKhalusova :D

> We embark on a journey that merges theoretical foundations, practical case studies, and ethical contemplations inherent in the landscape of computer vision. From exploring real-world examples like the ImageNet Roullete case study to evaluating biases in AI models recognizing "Gay Face", Twitter's Saliency Algorithm and similar case studies, this unit delves deep into understanding, assessing, and mitigating biases in computer vision systems.

Reviewer 5: same, small typo in "Roulette"

> By the conclusion of this unit, you'll have gained insights into recognizing biases, evaluating them in CV models, and employing strategies to mitigate these biases effectively. Additionally, you’ll explore the commendable efforts of HuggingFace in championing ethical practices within AI, providing a roadmap towards responsible and transparent AI development.

Reviewer: ```suggestion
By the conclusion of this unit, you'll have gained insights into recognizing biases, evaluating them in CV models, and effectively employing strategies to mitigate these biases. Additionally, you'll explore Hugging Face's commendable efforts to champion ethical practices within AI, providing a roadmap towards responsible and transparent AI development.
```

Author: Thank you merve, I made the changes 🤗

Join us as we navigate the intricate intersections of ethics and bias in the realm of computer vision, equipping ourselves to contribute ethically and responsibly to the future of AI and society.

> Let's shape the AI that is not only smart, but fair and responsible 

Reviewer 5: Maybe you can add a full stop at the end of the sentence here
